.devcontainer/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.145.1/containers/ubuntu/.devcontainer/base.Dockerfile
+
+# [Choice] Ubuntu version: bionic, focal
+ARG VARIANT="focal"
+FROM mcr.microsoft.com/vscode/devcontainers/base:0-${VARIANT}
+
+# Install build tools here
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y -qq install --no-install-recommends \
+    clang-format clang-tidy clang-tools clang clangd \
+    libc++-dev libc++1 libc++abi-dev libc++abi1 \
+    libclang-dev libclang1 liblldb-dev libllvm-ocaml-dev \
+    libomp-dev libomp5 lld lldb llvm-dev llvm-runtime llvm \
+    python-clang python3 python3-pip cmake ninja-build build-essential \ 
+    && pip3 install conan
+
+# Install additional tools here
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
```

Comment:
What about IWYU and selecting specific gcc and clang versions that current root Dockerfile has in the main branch?